### DISCUSSION TITLE
Dependabot and google protobuf

### DISCUSSION THREAD

**Discussion by xsavvyx** [dependabot/dependabot-core#9001] [Q&A]
Hi! We seem to be having a problem with dependabot updating our google-protobuf when it is updating other gems. 
Here is an example with its suggestion for updates surrounding nokogiri. I have been searching to see if others are having this issue, but have not found anything. Please let me know if you have insight.

<img width="1011" alt="Screenshot 2024-02-06 at 3 38 21 PM" src="https://github.com/dependabot/dependabot-core/assets/132275580/bd070c8a-5a02-4e83-8c07-761a766fa268">

**Comment by yeikel** [comment]
Can you create an issue and share more context? Typically discussions are not for this
